### PR TITLE
more cleanup

### PR DESCRIPTION
**Pull request by danrubel**
* remove BufferedProcess and use the one in atom.dart instead
* remove duplicate methods and use string_utils library in atom.dart
* update dartino:create-new-project
* remove promptUser and env and use the ones in atom.dart instead

@devoncarew

### CODE REVIEW

File: lib/atom_utils.dart [hunk 1]
```diff
@@ -21,73 +17,6 @@ import 'state.dart';
...
-  }));
-
-  Panel panel = atom.workspace.addModalPanel(item: element, visible: true);
-
-  Timer.run(() {
-    // There's a bug in Dart's JS interop where we can't see custom elements
-    // well. In order to work around it, we get the raw JS object, wrap it in a
-    // JsObject, pass that to a semantic wrapper around TextEditorElement, and
-    // then call `focused()`.
-    JsObject obj = new JsObject.fromBrowserObject(uncrackDart2js(editorElement));
```
Comment (devoncarew) [inline lib/atom_utils.dart]: We'll need to migrate this fix into atom.dart (I'm happy to take care of it).

**Comment by devoncarew** [comment]
:+1:  lgtm

File: lib/atom_utils.dart (hunk 1 above)
Comment (danrubel) [inline lib/atom_utils.dart]: Good eyes. Yes, please do so.

**Comment by danrubel** [comment]
Merged https://github.com/dart-atom/dartlang/commit/188f3045c5a005e9b771dc05570d52e4d9e28fa2

**Pull request closed without merging**